Getting-Started/Backoffice/Property-Editors/Built-in-Property-Editors/User-Picker/index.md

```diff
@@ -53,3 +53,43 @@ Please note that getting the Value of the property will return the user ID - pro
 }
 ```
 
+## Add values programmatically
+
+See the example below to see how a value can be added or changed programmatically. To update a value of a property editor you need the [Content Service](../../../../../Reference/Management/Services/ContentService/index.md).
+
+```csharp
+@{
+    // Get access to ContentService
+    var contentService = Services.ContentService;
+
+    // Create a variable for the GUID of the page you want to update
+    var guid = new Guid("796a8d5c-b7bb-46d9-bc57-ab834d0d1248");
+    
+	// Get the page using the GUID you've defined
+    var content = contentService.GetById(guid); // ID of your page
+	
+	// Set the value of the property with alias 'userPicker'
+    content.SetValue("userPicker", 1);
```

Comment:
The ´1´ in this line of code is the ID of a specific user, right?
Perhaps that should be explained in the comments or before/after the code snippet? 🤔 
What do you think?

Reply:
@sofietoft '1' is de specific user indeed. I changed it to -1 because it always exists. Also added a comment in code :-)